plugins/FlashSales/src/EasyAbp.EShop.Plugins.FlashSales.Domain.Shared/EasyAbp/EShop/Plugins/FlashSales/FlashSaleResults/FlashSaleResultFailedReason.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace EasyAbp.EShop.Plugins.FlashSales.FlashSaleResults;
+
+public static class FlashSaleResultFailedReason
+{
+    public const string InsufficientInventory = nameof(InsufficientInventory);
```

Comment:
Since renaming the property could cause an implicit breaking change, change to `public const string InsufficientInventory = "InsufficientInventory";` is better.